lib/archethic/bootstrap.ex

```diff
@@ -212,10 +214,44 @@ defmodule Archethic.Bootstrap do
     Sync.publish_end_of_sync()
     SelfRepair.start_scheduler()
 
+    if P2P.authorized_node?() do
```

Comment:
Here you should also controll if node is available

Reply:
noted!